worker/src/test/java/gov/cms/ab2d/worker/processor/JobProcessorIntegrationTest.java

```diff
@@ -135,10 +133,9 @@ private User createUser(Sponsor sponsor) {
     }
 
     private Contract createContract(Sponsor sponsor) {
-        final int anInt = random.nextInt(299);
         Contract contract = new Contract();
-        contract.setContractName("CONTRACT_0000" + anInt);
-        contract.setContractNumber("CONTRACT_0000" + anInt);
```

Comment:
This line in the test case was causing the contract number to not end in `0000` as a random number was being appended to the end of the contract number. I've removed this and now, it processes 10 patient records as expected